README.md

```diff
@@ -81,16 +81,34 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker
+execution time) after the page has been closed by the user. Periodic Background Sync extends it to
+potentially infinite time thereafter. Here are some privacy concerns:
```

Comment:
It's not infinite time after, it's still limited to a few minutes, just on more occasions.

Reply:
Sorry I wasn't clear. What I mean to say here is that the periodic task can be run forever (infinitely) unless the website unregisters or the browser decides heuristically that the task should be suspended. I'll try to make the wording clearer.